.mvn/jvm.config

```diff
@@ -0,0 +1,10 @@
+--add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED
```

Comment:
Please explain this change.

Reply:
Required on JDK 16 and newer due to [JEP 396: Strongly Encapsulate JDK Internals by Default](https://openjdk.java.net/jeps/396). Can't add comments in `jvm.config` file.

Reply:
Please add this to PR description, to let us easily come back to explanation later.

Reply:
Done.